src/app/public/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.ts

```diff
@@ -17,6 +17,10 @@ import {
 })
 export class SkyProgressIndicatorItemComponent {
 
+/**
+ * Specifies a header to display above the progress indicator.
```

Comment:
Is this right? I mean, according to the old docs, the `SkyProgressIndicatorTitleComponent` handles the header above the progress indicator. I think we need this description with that component. And I think this property should be something like `Specifies the step label to display for a step in the progress indicator.` I'm using `step label` to coincide with the terminology in the Anatomy graphic in the Design guidelines. Those guidelines also say that you can add step details, but I don't see a property for that. @Blackbaud-SteveBrush, does my description for this `tile` property look right to you? And how/where do you specify step details in the code?

Reply:
@blackbaud-johnly You are correct; the replacement description you provided looks good to me. I'm not sure what you mean with your last question, unless you mean the content added within the `sky-progress-indicator-item` component?
```
<sky-progress-indicator-item
  title="This item title"
>
  Details here.
</sky-progress-indicator-item>
```

Reply:
My second question is that if we have the `title` property on `sky-progress-indicator-item` to specify the step label, then how/where do users specify the step details? The design guidelines list it as an option, but I don't see a corresponding property in the development guidelines for implementing step details.

Reply:
@Blackbaud-SteveBrush following up with you on the pending question here. I updated the docs for the first item I raised is around how to display step details in the progress indicator. Based on the Anatomy and Options sections of the Design content, consumers can optionally display details under the step label:
![image](https://user-images.githubusercontent.com/7850105/83571173-4c2a3a00-a4f5-11ea-830c-d9a9dc917e4a.png)
We have the `title` property on `sky-progress-indicator-item` to specify the step label, but I don't see any property or other input for specifying the optional step details. Am I missing something?

Reply:
@Blackbaud-AlexKingman, pinging you on the above question too since I know that you've been helping Denise with these dev properties.

Reply:
There's not a property for it, but consumers can add aditional information by simply including it inside the `sky-progress-indicator-item` tag:
```
      <sky-progress-indicator-item
        title="Finished step"
      >
         Optional details about this step
      </sky-progress-indicator-item>
```

Reply:
OK. I see. I'd expected it to be a property alongside the `title` property. I'll serve the docs to check out the description for the item component to see if I was just overlooking the obvious or if we need to tweak its description. Thanks!

Reply:
Resolving this because I updated the description around this.